nntrainer/optimizers/plugged_optimizer.h

```diff
@@ -138,6 +138,16 @@ class PluggedOptimizer : public nntrainer::Optimizer {
    */
   void checkValidation() const override { optimizer_devel->checkValidation(); }
 
+  /**
+   * @brief     Add extra variables per weight if the optimizer needs any.
+   * @param dim Dimension of tensor to be added as a optimizer variable
+   * @retval #ML_ERROR_NONE Successful.
+   * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
+   */
+  virtual std::vector<TensorDim> addOptimizerVariable(TensorDim dim) override {
+    return std::vector<TensorDim>();
```

Comment:
this must be `return optimizer_devel->addOptimizerVariable(dim)`

Reply:
Done